src/icons/general/IconEdit.tsx

```diff
@@ -4,7 +4,7 @@ import AccessibleSVG from '../accessible-svg';
 
 import { Icon } from '../iconTypes';
 
-const IconEdit = ({ className = '', color = '#999', height = 14, title, width = 14 }: Icon) => (
+const IconEdit = ({ className = '', color = '#767676', height = 14, title, width = 14 }: Icon) => (
```

Comment:
I assume we had to change this everywhere? Since it's a prop, we could have just increased it for that one specific section, right?

Reply:
Chatted in person about this. It is something we want to update across the board. I'll be approving.

Reply:
In fact any lighter gray  in the page over white background should change for $bld-gray-62, in this case I had to put it right on the svg, because its not taking styles from anywhere else :(